src/repository/scriptRepository.js

```diff
@@ -0,0 +1,83 @@
+const path = require('path');
+const Promise = require('bluebird');
+const fs = Promise.promisifyAll(require('node:fs'));
+
+const db = require('./database.js');
+const { Character, parseTitle, parseLines, parseCharacters } = require('../playData.js');
+
+function getCharacters(title, actorId = -1) {
+  const values = [actorId, title];
+  const sql = `
```

Comment:
New sql query can now filter the characters for a particular actor instead of having to do it in js